README.md

```diff
@@ -1,21 +1,438 @@
+  }
+}
+```
+
+Worked like a charm! Nothing we didn't want, and our controller is back
+to a clear, non-messy state.
+
+### Rendering An Author
+
+What's missing that we had before? The author name. So how do we do
+that?
+
+Because AMS is modeled after the way Rails handles models and
+controllers, rather than build serialization of the author into the
+post, as we have in the past, we need to create a new
+`AuthorSerializer`.
+
+`rails g serializer author`
+
+And add the author name to the list of attributes:
+
+```ruby
+class AuthorSerializer < ActiveModel::Serializer
+  attributes :id, :name
+end
+```
+
+Now to test this out, let's modify our `authors_controller#show` action
+to handle a JSON request:
+
+```ruby
+class AuthorsController < ApplicationController
+  def show
+    @author = Author.find(params[:id])
+    respond_to do |f|
+      f.html { render :show }
+      f.json { render json: @author }
+    end
+  end
+end
+```
+
+And load up `/authors/id.json`. We should see something that looks like
+this:
+
+```javascript
+{
+  author: {
+    id: 1,
+    name: "Stephen King"
+  }
+}
+```
+
+But how to we add the author name to our post JSON?
+
+Again, we lean on those Rails conventions. If we add a `has_one :author`
+to our `PostSerializer`:
+
+```ruby
+class PostSerializer < ActiveModel::Serializer
+  attributes :id, :title, :description
+  has_one :author
+end
+```
+
+**Top-tip:** You might notice that we're using `has_one` in our
+serializer, when in the model, the post -> author relationship is
+actually a `belongs_to`. This can be confusing at first, but AMS is only
+concerned with the number of the relationship, not the direction. So it
+only knows `has_one` and `has_many`. This is because it's not describing
+a data/model relationship, but the relationship as defined by the JSON.
+
+Reload `/posts/id.json` we will now see our author information.
+
+If we load `/posts/1` and try next, it doesn't work! What happened? Look again at the JSON from `/posts/1.json`:
+
+```javascript
+{
+  post: {
+    id: 1,
+    title: "A Blog Post By Stephen King",
+    description: "This is a blog post by Stephen King. It will probably be a movie soon.",
+    author: {
+      id: 1,
+      name: "Stephen King"
+    }
+  }
+}
+```
+
+What's changed?
+
+We now have a `root` node of `post: {` in our JSON, instead of just the
+values. This is the default because it more accurately describes the
+JSON response. All we have to do to fix our `show` page is to add
+checking the `["post"]` node to our Javascript:
+
+```erb
+# posts/show.html.erb
+# ...
+<script type="text/javascript" charset="utf-8">
+$(function () {
+  $(".js-next").on("click", function() {
+    var nextId = parseInt($(".js-next").attr("data-id")) + 1;
+    $.get("/posts/" + nextId + ".json", function(data) {
+      // get post
+      var post = data["post"];
+      $(".authorName").text(post["author"]["name"]);
```

Comment:
Hm, fair enough. I think it'd be nice to transition to dot-notation at some point over the course of these READMEs and labs, simply because that's what students will encounter must of the time in the wild, wild jungles of JavaScript. But since this is kind a full-stack thing and it's probably a good idea to get them to understand that JS Objects and Ruby Hashes represent the same concept, I don't want to be super uptight about it.